src/components/ExchangeRateHistoryComponent/ExchangeRateHistoryComponent.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../styles/utils/variables";
+
+.history-rate-container {
+  display: block;
```

Comment:
If it's a 'div' element, then display: block is set by default (browser generic styles based on HTML standardized guidelines)

Reply:
Dear Radek, you are right. Good catch thanks